v1/openings/views/opening.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+from rest_framework import status
+from rest_framework.mixins import (
+    CreateModelMixin,
+    DestroyModelMixin,
+    ListModelMixin,
+    RetrieveModelMixin,
+    UpdateModelMixin
+)
+from rest_framework.response import Response
+from rest_framework.viewsets import GenericViewSet
+
+from v1.third_party.rest_framework.permissions import IsStaffOrReadOnly
+from ..models import Opening
+from ..serializers import OpeningSerializer, OpeningSerializerWrite
+
+
+class OpeningViewSet(
+    CreateModelMixin,
+    DestroyModelMixin,
+    ListModelMixin,
+    RetrieveModelMixin,
+    UpdateModelMixin,
+    GenericViewSet,
+):
+    queryset = Opening.objects \
+        .prefetch_related('reports_to', 'responsibilities', 'skills') \
+        .order_by('created_date') \
+        .all()
+    serializer_class = OpeningSerializer
+    serializer_write_class = OpeningSerializerWrite
```

Comment:
To be able to return the response object in a different shape. When creating the opening we want to allow the user to pass in only the `pk` for the related objects but in the response it will be serialized as the full object to avoid an additional request by the FE.